index.html

```diff
@@ -4,9 +4,44 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:wght@100;300;400&family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400&family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Mobile version skeleton</title>
 </head>
 <body>
+    <header class="header">
+        <h1 class="nav_bar_text">My Logo</h1>
```

Comment:
- [ ] Please use the `a` tag instead of the `h1` tag. `hx` tags are meant for the main content **headings** or **subheadings** not for the logo, and it is semantically not correct which can result in *bad user experience*. You can take an example of a person and just assume that your website is bigger and have several pages and a user just lost somewhere and that person wants to go back to the home page then in that case apart from `Home` navigation this `logo` with the help of `a` tag makes that person work easier to securely come back on `home-page`.